.gitignore

```diff
@@ -2,6 +2,9 @@
 #
 # gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
 
+## macOS
+.DS_Store
+
```

Comment:
Why do you think it's better remove this? 🙂

Reply:
In fact, it prevents contributors to accidentally add the `.DS_Store` file, and to waste time asking them to remove it before merging. When everything else is ok, it's a little bit frustrating for everyone 😉